samples/MeteoControl/app/application.cpp

```diff
@@ -54,7 +54,7 @@ void init()
 	lcd.createChar(6, icon_cross);
 	lcd.createChar(7, icon_check);
 
-	WifiStation.config(ActiveConfig.NetworkSSID, ActiveConfig.NetworkPassword);
+	WifiStation.config(ActiveConfig.NetworkSSID.c_str(), ActiveConfig.NetworkPassword.c_str());
```

Comment:
That change breaks the compatibility with already existing code. Also there should be quantitative measurement that proves that this effort is worth it.

Isn't it possible to have similar improvements by changing ?

```
StationClass::config(String ssid, String password, bool autoConnectOnStartup /* = true*/, bool save /* = true */)
```

to 

```
`StationClass::config(const String& ssid, const String& password, bool autoConnectOnStartup /* = true*/, bool save /* = true */) 
```